chrome/content/zotero/xpcom/reader.js

```diff
@@ -1986,6 +1987,17 @@ class Reader {
 				}
 			}
 		}
+		else if (type === 'setting') {
+			let id = ids[0];
+			let newCustomThemes = Zotero.SyncedSettings.get(Zotero.Libraries.userLibraryID, 'readerCustomThemes') ?? [];
```

Comment:
Might as well move this within the conditional

Reply:
good point, fixed!